### PR TITLE
Add 2019 Kenya Census Data

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- [x] Add kenya census data

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot from 2019-11-28 13-46-01
Screenshot from 2019-11-28 13-45-43
Screenshot from 2019-11-28 13-45-02

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/profiles/census.py [hunk 1]
```diff
@@ -43,6 +43,30 @@
...
+                    'title': '2019 Kenya Population and Housing Census Volume I',
+                },
+            },
+             'population_sex_2019': {
+                'source': {
+                    'link': 'https://www.knbs.or.ke/?wpdmpro=2019-kenya-population-and-housing-census-volume-i-population-by-county-and-sub-county',
+                    'title': '2019 Kenya Population and Housing Census Volume I',
+                },
+            },
+             'subcounty_population_sex_2019': {
```
Comment (kilemensi) [inline takwimu/profiles/census.py]: 1. We don't have subcounties in Takwimu/HURUmap just yet @KhadijaMahanga, correct?
2. Any reason why we're keeping these in a separate table and not the `population_sex_2019`?

Comment (KhadijaMahanga) [reply takwimu/profiles/census.py]: we don't have subcounty that's the reason they're in different tables... so `subcounty population sex 2019` table has subcounty as a field. Meaning its population for county per each subcounty per gender.

Comment (KhadijaMahanga) [reply takwimu/profiles/census.py]: @kilemensi

Comment (kilemensi) [reply takwimu/profiles/census.py]: Would it break anything if the data is in the same table @KhadijaMahanga? Or can we hold it until we add sub-counties?

Comment (kilemensi) [reply takwimu/profiles/census.py]: Ahh... I see we're using it as just _data_ at county level.
Population per subcounty 2019

:rocket: @KhadijaMahanga

Comment (DavidLemayian) [reply takwimu/profiles/census.py]: They're called "constituencies". 🙄

**Comment by KhadijaMahanga** [comment]
@kilemensi  column insteadof grouped_column chart for subcounty data

Screenshot from 2019-11-29 11-29-59

**Comment by kilemensi** [comment]
Much cleaner for me @KhadijaMahanga! :rocket:

**Comment by DavidLemayian** [comment]
But why do those charts look so much better than I remember?! 😍 🚀

**Pull request merged**